src/simulation/TxGenerator.cpp

```diff
@@ -224,70 +224,63 @@ TxGenerator::createAccounts(uint64_t start, uint64_t count, uint32_t ledgerNum,
 
 TransactionFrameBasePtr
 TxGenerator::createTransactionFramePtr(
-    TxGenerator::TestAccountPtr from, std::vector<Operation> ops, bool pretend,
+    TxGenerator::TestAccountPtr from, std::vector<Operation> ops,
     std::optional<uint32_t> maxGeneratedFeeRate)
 {
     auto txf = transactionFromOperations(
         mApp, from->getSecretKey(), from->nextSequenceNumber(), ops,
         generateFee(maxGeneratedFeeRate, ops.size()));
-    if (pretend)
-    {
-        Memo memo(MEMO_TEXT);
-        memo.text() = std::string(28, ' ');
-        txbridge::setMemo(txf, memo);
 
-        txbridge::setMinTime(txf, 0);
-        txbridge::setMaxTime(txf, UINT64_MAX);
+    return txf;
+}
 
-        txbridge::getSignatures(txf).clear();
-        txf->addSignature(from->getSecretKey());
-    }
+TransactionFrameBasePtr
+TxGenerator::createTransactionFramePtr(
+    TxGenerator::TestAccountPtr from, std::vector<Operation> ops,
+    std::optional<uint32_t> maxGeneratedFeeRate, uint32_t byteCount)
+{
+    auto txf = paddedTransactionFromOperations(
+        mApp, from->getSecretKey(), from->nextSequenceNumber(), ops,
+        generateFee(maxGeneratedFeeRate, ops.size()), byteCount);
 
     return txf;
 }
 
 std::pair<TxGenerator::TestAccountPtr, TransactionFrameBasePtr>
 TxGenerator::paymentTransaction(uint32_t numAccounts, uint32_t offset,
                                 uint32_t ledgerNum, uint64_t sourceAccount,
-                                uint32_t opCount,
+                                uint32_t byteCount,
                                 std::optional<uint32_t> maxGeneratedFeeRate)
 {
     TxGenerator::TestAccountPtr to, from;
     uint64_t amount = 1;
     std::tie(from, to) =
         pickAccountPair(numAccounts, offset, ledgerNum, sourceAccount);
     vector<Operation> paymentOps;
-    paymentOps.reserve(opCount);
-    for (uint32_t i = 0; i < opCount; ++i)
-    {
-        paymentOps.emplace_back(txtest::payment(to->getPublicKey(), amount));
-    }
+    paymentOps.emplace_back(txtest::payment(to->getPublicKey(), amount));
 
-    return std::make_pair(from,
-                          createTransactionFramePtr(from, paymentOps, false,
-                                                    maxGeneratedFeeRate));
+    return std::make_pair(from, createTransactionFramePtr(from, paymentOps,
+                                                          maxGeneratedFeeRate,
+                                                          byteCount));
 }
 
 std::pair<TxGenerator::TestAccountPtr, TransactionFrameBasePtr>
 TxGenerator::manageOfferTransaction(uint32_t ledgerNum, uint64_t accountId,
-                                    uint32_t opCount,
+                                    uint32_t byteCount,
```

Comment:
I think you can remove the `manageOfferTransaction` function entirely now that `MIXED_CLASSIC` mode is gone.